packages/react-widgets/src/widgets/FormulaWidget.js

```diff
@@ -48,12 +53,16 @@ function FormulaWidget({
 
   return (
     <WrapperWidgetUI title={title} isLoading={isLoading} {...wrapperProps}>
-      <FormulaWidgetUI
-        data={data}
-        formatter={formatter}
-        unitBefore={true}
-        animation={animation}
-      />
+      {isDroppingFeatures ? (
+        <NoDataAlert severity="warning" body="Some rows have been filtered at this zoom level. Zoom in to ensure you see all rows in the map." />
```

Comment:
I'm not sure about the body property and this should be configurable (for all the widgets). I think some rows have been "filtered out" and if you zoom in, you don't see all rows in the map, what you do is to ensure that no features in the current viewport have been filtered out. I'd also refer to features/elements instead of rows.

Reply:
Copy from @ernesmb are here: https://app.shortcut.com/cartoteam/story/222812/design-widget-state-when-feature-dropping-is-happening. 

Let's validate those!

Reply:
I understand this is the copy that Ernesto wants for Builder and that's fine, but the copy itself needs to be configurable. We can have a default value and then have a way to override this default value by the applications, in the same way that NoDataAlert component works. I think the current implementation does not meet this requirement. I already mentioned that before this got implemented.

Reply:
I agree with Borja. In others widgets, we have the noDataAlertProps but they are being overwritten when isDroppingFeatures is true:

```
<NoDataAlert {...noDataAlertProps} {...(isDroppingFeatures ? { severity: 'warning', body: 'Some rows have been filtered at this zoom level. Zoom in to ensure you see all rows in the map.' } : {})}/>
```

> I'd also refer to features/elements instead of rows.

This for sure!